.gitignore

```diff
@@ -6,3 +6,4 @@ target
 project/.sbt
 # if you are here to add your IDE's files please read this instead:
 # https://stackoverflow.com/questions/7335420/global-git-ignore#22885996
+.DS_Store
```

Comment:
This needs to go in your personal gitignore file, see [here](https://stackoverflow.com/questions/7335420/global-git-ignore#22885996) for instructions.

Reply:
how to delete from this pull?

Reply:
Just delete the line and commit 😃

Reply:
Yes, just delete this single line from the `.gitignore` file (you can add it to your personal one).